R/permillgraphfunc.R

```diff
@@ -21,7 +21,7 @@ permillgraph <- function(state, all, capita) {
     if(state == "United States"){
       if (all) {
         matplot(
-          2000:(as.integer(format(Sys.Date(), "%Y"))-2),
+          2000:2017,
```

Comment:
it looks like you're hard coding to 2017 here -- i think we want to preserve the open end date.